packages/appstore/src/components/app-launcher/app-launcher.scss

```diff
@@ -13,8 +13,20 @@
     }
 
     &__button {
-        height: fit-content;
         margin-left: auto;
+        display: flex;
+        flex-direction: row;
+        justify-content: center;
+        align-items: center;
+        padding: 6px 16px;
```

Comment:
padding: 0.6rem 1.6rem

Reply:
We are not using this in this PR

Reply:
if you are not using it here,you can remove them, if not change it to `rem`